### ISSUE TITLE
[enterprise-4.12] Issue in file networking/network_observability/network-observability-overview.adoc

### ISSUE THREAD

**Issue by eranra** [openshift/openshift-docs#54810]
### Which section(s) is the issue in?

Updated text for "Dependency of Network Observability Operator"

@stleerh @skrthomas 

### What needs fixing?

```
The Network Observability Operator requires the following components to be installed:
•	Loki: Loki is the backend persistent log store for NetFlow data collected by the Network Observability Operator. It is recommended to install Loki by installing the Red Hat Loki Operator. For details, refer to [LokiStack deployment instructions](https://docs.openshift.com/container-platform/4.12/logging/cluster-logging-loki.html#logging-loki-deploy_cluster-logging-loki) under the Logging sub-section
```

**Comment by lmurthyrh** [comment]
@skrthomas  - For your attention

**Comment by skrthomas** [comment]
Closed with GH#54809
